### PR TITLE
feat: use entrust ssl certs in dev, test, prod

### PR DESCRIPTION
**Pull request by wenzowski**
For #378

Cronjob will need to be manually run

### CODE REVIEW

**Comment by wenzowski** [comment]
corresponding environment vars configured in dev/test/prod

File: helm/app/templates/routes/secureRoute.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{{- $route := (lookup "route.openshift.io/v1" "Route" .Release.Namespace "your-route" ) }}
```
Comment (matthieu-foucault) [inline helm/app/templates/routes/secureRoute.yaml:1]:
```suggestion
{{- $route := (lookup "route.openshift.io/v1" "Route" .Release.Namespace "{{ template "ccbc.fullname" . }}-secure" ) }}
```
not sure about the syntax, I hardcoded this in cif...

File: helm/app/values.yaml [hunk 2]
```diff
@@ -129,3 +131,13 @@ pgBouncer:
...
+
+certbot:
+  image:
+    tag: 1.0.0
+    pullPolicy: IfNotPresent
+  certbot:
+    email: '' # The value must be passed in via the deploy script
+    server: '' # The value must be passed in via the deploy script
+  cron:
+    suspend: true # until certs are issued
```
Comment (matthieu-foucault) [inline helm/app/values.yaml:143]: are you going to manually run the job?

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]

File: helm/app/values.yaml (hunk 2 above)
Comment (wenzowski) [reply helm/app/values.yaml:143]: Yes, intending to manually trigger the cronjob and PR to remove this once all 3 certs are issued

File: helm/app/templates/routes/secureRoute.yaml (hunk 1 above)
Comment (wenzowski) [reply helm/app/templates/routes/secureRoute.yaml:1]: whoopsie, tyvm

**Review by matthieu-foucault — APPROVED** [review]
:shipit:

**Pull request merged**